src/overlays/actors/ovl_En_Tru/z_en_tru.c

```diff
@@ -24,6 +24,26 @@ s32 func_80A87DC0(Actor* thisx, PlayState* play);
 void func_80A87FD0(EnTru* this, PlayState* play);
 void func_80A881E0(EnTru* this, PlayState* play);
 
+typedef enum {
+    /* 0x00 */ KOUME_ANIM_INJURED_LYING_DOWN1,
+    /* 0x01 */ KOUME_ANIM_INJURED_LYING_DOWN2,
```

Comment:
Would want to change these two if you change the animation name. Also, can you tell why one of these would be used instead of the other?

Reply:
The difference between these two would be the morph frames (renamed them so the ones with morph frames end with the word morph), which AFAIK is for a smoother transition between animations, but I'm not 100% sure.